README.md

```diff
@@ -257,6 +257,37 @@ Hello Someusername
 ```
 
 
+### Output with Color and Bold letters
+If you want to change the output such as coloring and bold letters, you can do as follows(example is using [fatih/color](https://github.com/fatih/color)).
+
+```go
+import "github.com/fatih/color"
+
+cyan := color.New(color.FgCyan).SprintFunc()
+yellow := color.New(color.FgYellow).SprintFunc()
+boldRed := color.New(color.FgRed, color.Bold).SprintFunc()
+shell.AddCmd(&ishell.Cmd{
+    Name: "color",
+    Help: "color print",
+    Func: func(c *ishell.Context) {
+        c.Print(cyan("cyan\n"))
+        c.Println(yellow("yellow"))
+        c.Printf("%s\n", boldRed("bold red"))
+    },
+}
+```
```

Comment:
To save space for the README considering full example is in `example/main.go`, I would suggest we use a single color. You can change it to something like
```go
func(c *ishell.Context) {
    yellow := color.New(color.FgYellow).SprintFunc()
    c.Println(yellow("This output is in yellow"))
}
```